src/components/WwwFrame/TheHeader.vue

```diff
@@ -304,15 +305,16 @@
 							</router-link>
 
 							<!-- Log in Link -->
-							<router-link
-								:to="loginUrl"
+							<kv-button
+								variant="secondary"
 								v-show="isVisitor"
+								class="header__login__button tw-bg-white"
+								:to="loginUrl"
 								data-testid="header-log-in"
-								class="header__button header__log-in"
```

Comment:
By removing this class `header__button` you've lost the flex classes that would control vertical placement of this item. You should re-apply some of those classes to get your vertical placement either here on the element or below using your class.